python/src/trezorlib/debuglink.py

```diff
@@ -599,28 +651,30 @@ def _verify_responses(cls, expected, actual):
                 output.append(textwrap.indent(protobuf.format_message(act), "    "))
                 raise AssertionError("\n".join(output))
 
-    def mnemonic_callback(self, _):
+    def mnemonic_callback(self, _) -> str:
         word, pos = self.debug.read_recovery_word()
         if word != "":
```

Comment:
i guess this should be `if word` and `if pos` -- no point checking the empty string but letting a `None` through

Reply:
Done in b70acaf